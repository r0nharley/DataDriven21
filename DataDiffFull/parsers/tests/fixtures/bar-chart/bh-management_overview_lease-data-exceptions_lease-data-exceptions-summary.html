<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="1454" height="431">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-6xri21s-1">
            <rect x="0" y="0" width="385" height="1045" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-6xri21s-2">
            <rect x="0" y="0" width="385" height="1045" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="1454" height="431" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="399" y="10" width="1045" height="385"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 399 87.5 L 1444 87.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 399 164.5 L 1444 164.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 399 241.5 L 1444 241.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 399 318.5 L 1444 318.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 399 395.5 L 1444 395.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 399 10.5 L 1444 10.5" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 398.5 10 L 398.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 446.5 10 L 446.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 493.5 10 L 493.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 541.5 10 L 541.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 588.5 10 L 588.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 636.5 10 L 636.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 683.5 10 L 683.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 731.5 10 L 731.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 778.5 10 L 778.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 826.5 10 L 826.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 873.5 10 L 873.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 921.5 10 L 921.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 968.5 10 L 968.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1016.5 10 L 1016.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1063.5 10 L 1063.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1111.5 10 L 1111.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1158.5 10 L 1158.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1206.5 10 L 1206.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1253.5 10 L 1253.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1301.5 10 L 1301.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1348.5 10 L 1348.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1396.5 10 L 1396.5 395" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 1444.5 10 L 1444.5 395" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="399" y="10" width="1045" height="385"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 87.5 L 389 87.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 164.5 L 389 164.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 241.5 L 389 241.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 318.5 L 389 318.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 395.5 L 389 395.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 399 9.5 L 389 9.5" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 398.5 10 L 398.5 395"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 399 395 L 1444 395"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker highcharts-series-hover" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-2)">
            <rect x="360.5" y="60.5" width="8" height="985" fill="rgb(135,243,110)" class="highcharts-point highcharts-point-hover" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-series-hover" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-2)">
            <rect x="274.5" y="1026.5" width="8" height="19" fill="#06e5ff" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-2 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-2)">
            <rect x="188.5" y="987.5" width="8" height="58" fill="#fbb755" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-2 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-3 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-2)">
            <rect x="102.5" y="1045.5" width="8" height="0" fill="#218A8C" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-3 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-4 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="url(https://secure.rentlytics.com/bh-management/bi/overview-lease-data-exceptions/visualization#highcharts-6xri21s-2)">
            <rect x="15.5" y="997.5" width="8" height="48" fill="#7efb62" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-4 highcharts-bar-series highcharts-color-undefined " transform="translate(1444,395) rotate(90) scale(-1,1) scale(1 1)" width="1045" height="385" clip-path="none"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(399,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(985,9)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-1 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(399,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(19,95)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-2 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(399,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(58,181)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-3 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(399,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(0,267)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-4 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(399,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(48,354)"></g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="384" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="53" opacity="1">
            <tspan>Occupied Units with Lease Expiration Dates &gt; 7 Days in the Past</tspan>
        </text>
        <text x="384" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="130" opacity="1">
            <tspan>Vacant Rented Units with Lease Hold Period &gt; 2 Months</tspan>
        </text>
        <text x="384" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="207" opacity="1">
            <tspan>Any Units with Lease Terms &gt; 25 Months</tspan>
        </text>
        <text x="384" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="284" opacity="1">
            <tspan>Occupied, Missing Move-In Date</tspan>
        </text>
        <text x="384" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:470px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="361" opacity="1">
            <tspan>Vacant Unrented Units with Lease Dates</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="399" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">0</text>
        <text x="446.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">100</text>
        <text x="494" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">200</text>
        <text x="541.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">300</text>
        <text x="589" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">400</text>
        <text x="636.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">500</text>
        <text x="684" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">600</text>
        <text x="731.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">700</text>
        <text x="779" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">800</text>
        <text x="826.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">900</text>
        <text x="874" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,000</text>
        <text x="921.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,100</text>
        <text x="969" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,200</text>
        <text x="1016.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,300</text>
        <text x="1064" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,400</text>
        <text x="1111.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,500</text>
        <text x="1159" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,600</text>
        <text x="1206.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,700</text>
        <text x="1254" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,800</text>
        <text x="1301.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">1,900</text>
        <text x="1349" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">2,000</text>
        <text x="1396.5" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">2,100</text>
        <text x="1432.125" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:23.75;text-overflow:ellipsis;" text-anchor="middle" transform="translate(0,0)" y="414" opacity="1">
            <tspan>2,…</tspan>
            <title>2,200</title>
        </text>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(919,1)" opacity="1" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 459 0 C 459 0 459 0 459 0 L 459 23 465 29 459 35 459 102 C 459 102 459 102 459 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>