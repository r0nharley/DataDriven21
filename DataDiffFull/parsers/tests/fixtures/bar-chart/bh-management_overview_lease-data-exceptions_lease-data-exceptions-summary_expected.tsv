Occupied Units with Lease Expiration Dates > 7 Days in the Past	Data	2073.68421053
Vacant Rented Units with Lease Hold Period > 2 Months	Data	40.0
Any Units with Lease Terms > 25 Months	Data	122.105263158
Occupied, Missing Move-In Date	Data	0.0
Vacant Unrented Units with Lease Dates	Data	101.052631579